packages/dropdown/src/styledComponents/DropdownWrapper.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+export default styled.div`
+  font-family: lato, sans-serif;
```

Comment:
This is a normally a sign that the Text component could've been used...

Maybe we could check in the DropDownItem if the children is a text node and wrap it in the proper Text component (with the proper size and color that is expected as default on menus), but probably over-engineering at the moment, so OK